src/components/Hospital.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class Hospital extends Component {
+    render() {
+        const { id, name, createdAt, updatedAt } = this.props.hospital;
+        return (
+            <tr>
+                <td><Link to={{ pathname: "/editHospital", state: {hospital: this.props.hospital}}}>{id}</Link></td>
+                <td>{name}</td>
```

Comment:
Right now, we're hard coding in the properties we want displayed (id, name, createdAt, updatedAt), but a better way might be to just map all the properties of `this.props.hospital`.

Reply:
so maybe something like:

`{this.props.hospital.map(property => <td>{property}</td>)}`

Reply:
sorry i'm not sure how to directly suggest changes on the code

Reply:
I'm not sure if this is the intended functionality though - let me know what you think.

Reply:
yeah this is with my limited knowledge of `react-router`, but you're right, spreading the properties on the component would be better in general